src/aspire/utils/filters.py

```diff
@@ -30,7 +30,9 @@ def evaluate(self, omega):
         if omega.ndim == 1:
             ensure(self.radial, f'Cannot evaluate a non-radial filter on 1D input array.')
         elif omega.ndim == 2:
-            ensure(omega.shape[0] == self.dim, f'Omega must be of size {self.dim} x n')
+            if self.dim:
```

Comment:
can we save a line and just move this one level up (using `and`). Sorry to be a pain, but extra lines like these are a big eyesore.

Reply:
Yes. Done on commit 82c37af. Will rebase this branch and remove some intermediate commits.